extensions/amp-social-share/amp-social-share.md

```diff
@@ -61,6 +61,36 @@ When you want to pass parameters to the share endpoint, you can specify `data-pa
 
 Linkedin is one of the pre-configured providers, so you do not need to provide the `data-share-endpoint` attribute.
 
+### Standalone use outside valid AMP documents
+
+Bento AMP allows you to use AMP components in non-AMP pages without needing to commit to fully valid AMP. You can take these components and place them in implementations with frameworks and CMSs that don't support AMP. Read more in our guide `[Use AMP components in non-AMP pages](https://amp.dev/documentation/guides-and-tutorials/start/bento_guide/)`.
+
+#### Example
+
+The example below demonstrates `amp-social-share` component in standalone use.
+
+```html
+<head>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <link rel="stylesheet" type="text/css" href="https://cdn.ampproject.org/v0/amp-social-share-1.0.css">
+  <script async custom-element="amp-social-share" src="https://cdn.ampproject.org/v0/amp-social-share-1.0.js"></script>
+</head>
+<amp-social-share type="email"></amp-social-share>
+<amp-social-share type="twitter"></amp-social-share>
+<amp-social-share type="tumblr"></amp-social-share>
+<amp-bento-component>
+```
```

Comment:
```suggestion
[example preview="top-frame" playground="false"]
```html
<head>
  <script async src="https://cdn.ampproject.org/v0.js"></script>
  <link rel="stylesheet" type="text/css" href="https://cdn.ampproject.org/v0/amp-social-share-1.0.css">
  <script async custom-element="amp-social-share" src="https://cdn.ampproject.org/v0/amp-social-share-1.0.js"></script>
</head>
<amp-social-share type="email"></amp-social-share>
<amp-social-share type="twitter"></amp-social-share>
<amp-social-share type="tumblr"></amp-social-share>
<amp-bento-component>
```
[/example]
```

Reply:
Updated in latest, thanks!